src-plugins/itkFilters/itkFiltersInvertProcess_p.h

```diff
@@ -41,9 +42,19 @@ class itkFiltersInvertProcessPrivate : public itkFiltersProcessBasePrivate
         typedef itk::Image< PixelType, 3 > ImageType;
         typedef itk::InvertIntensityImageFilter< ImageType, ImageType >  InvertFilterType;
         typename InvertFilterType::Pointer invertFilter = InvertFilterType::New();
-    
+
+        // compute maximum intensity of image
+        typedef itk::MinimumMaximumImageCalculator< ImageType > MinMaxFilterType;
+        typename MinMaxFilterType::Pointer maxFilter = MinMaxFilterType::New();
+
+        maxFilter->SetImage( dynamic_cast<ImageType *> ( ( itk::Object* ) ( input->data() ) ) );
+        maxFilter->Compute();
+        PixelType maximum = maxFilter->GetMaximum();
+        PixelType minimum = maxFilter->GetMinimum();
+
         invertFilter->SetInput ( dynamic_cast<ImageType *> ( ( itk::Object* ) ( input->data() ) ) );
-        
+        invertFilter->SetMaximum(maximum + minimum);
```

Comment:
This was a tricky part, as the documentation for this filter is sparse... so I just tried different possibilities.

Let's say you have an image in [x, y]. By setting only max, you get [0, y - x].
By setting  min + max you get the correct [x, y]....